selfdrive/ui/ui.h

```diff
@@ -42,12 +42,16 @@ typedef enum UIStatus {
   STATUS_DISENGAGED,
   STATUS_OVERRIDE,
   STATUS_ENGAGED,
+  STATUS_LAT_ONLY,
+  STATUS_LONG_ONLY,
 } UIStatus;
 
 const QColor bg_colors [] = {
   [STATUS_DISENGAGED] = QColor(0x17, 0x33, 0x49, 0xc8),
   [STATUS_OVERRIDE] = QColor(0x91, 0x9b, 0x95, 0xf1),
   [STATUS_ENGAGED] = QColor(0x17, 0x86, 0x44, 0xf1),
+  [STATUS_LAT_ONLY] = QColor(0x00, 0xc8, 0xc8, 0xf1),
+  [STATUS_LONG_ONLY] = QColor(0x96, 0x1C, 0xA8, 0xf1),
```

Comment:
```suggestion
  [STATUS_LONG_ONLY] = QColor(0xFF, 0x1C, 0xE3, 0xF1),
```
from:
<img width="81" alt="image" src="https://github.com/user-attachments/assets/9a2be4ee-df44-427b-9014-337be6b059a7" />

to:
<img width="143" alt="image" src="https://github.com/user-attachments/assets/6a93d177-24b5-48fd-8f14-7c29e47f636f" />

Reply:
`QColor` with 4 arguments is ARGB. This is the colors with the current hex assignments:
![output](https://github.com/user-attachments/assets/423a4fcf-db2d-4bfd-8e4c-66058bdf804c)